example/cluster-machine-template-simple.yaml

```diff
@@ -277,7 +277,7 @@ spec:
           name: "{{ ds.meta_data.local_hostname }}"
           kubeletExtraArgs:
             cloud-provider: external
-            provider-id: aws:///'{{ ds.meta_data.instance_id }}'
+            provider-id: aws://eu-west-2a/'{{ ds.meta_data.instance_id }}'
```

Comment:
"eu-west-2a", it should be retrieve from the metadata.
The following command will give you the subregion:
```
curl  http://169.254.169.254/latest/meta-data/placement/availability-zone
```

You should test if ```{{ ds.meta_data.placement.availability_zone }}``` gives you what you want

Reply:
Ok done thanks.